modules/ROOT/pages/access-logging.adoc

```diff
@@ -72,17 +70,17 @@ The following table lists the available log format options.
 |The local IP address
 
 |%b
-|The response size in bytes, excluding the headers
+|The response size in bytes excluding the headers
 
 |%B
-|The response size in bytes, excluding the headers
-If no value is found, `0` is printed instead of `-`.
+|The response size in bytes excluding the headers
+`0` is printed instead of `-` if no value is found.
```

Comment:
i think it's more clear the other way around:

If no value is found, `0` is printed instead of `-`.